app/reducers/users.ts

```diff
@@ -97,3 +97,12 @@ export const selectUserWithGroups = createSelector(
     };
   },
 );
+
+export const selectUsersWithAchievementScore = createSelector(
+  selectUserEntities,
+  (userEntities) => {
+    return Object.values(userEntities).filter(
+      (user) => user.achievementScore != null,
```

Comment:
might want to do 
```js
(user) => !!user.achievementScore,
```

so that we catch `undefined` and `0` as well as `null`

Reply:
This is fine; it catches both `undefined` and `null` and there are no user entities of `0`